### PR TITLE
Update PartiallySupportedFlowKeywords.cpp All STOP Version

### PR DESCRIPTION
**Pull request by OPMUSER**

### CODE REVIEW

File: opm/simulators/utils/PartiallySupportedFlowKeywords.cpp [hunk 1]
```diff
@@ -85,36 +85,36 @@ partiallySupported()
...
             },
          },
          {
             "GCONPROD",
             {
-               {2,{false, allow_values<std::string> {"NONE", "FLD", "ORAT", "WRAT", "GRAT", "LRAT", "RESV"}, "GCONPROD(TARGET): valid option should be FLD/ORAT/WRAT/GRAT/LRAT or RESV – will STOP"}}, // CONTROL_MODE
-               {11,{false, allow_values<std::string> {}, "GCONPROD(ACTWAT): water violation procedure not implemented – will STOP"}}, // WATER_EXCEED_PROCEDURE
-               {12,{false, allow_values<std::string> {}, "GCONPROD(ACTGAS): gas violation procedure not implemented – will STOP"}}, // GAS_EXCEED_PROCEDURE
-               {13,{false, allow_values<std::string> {}, "GCONPROD(ACTLIQ): liquid violation procedure not implemented – will STOP"}}, // LIQUID_EXCEED_PROCEDURE
+               {2,{false, allow_values<std::string> {"NONE", "FLD", "ORAT", "WRAT", "GRAT", "LRAT", "RESV"}, "GCONPROD(TARGET): valid option should be NONE/FLD/ORAT/WRAT/GRAT/LRAT or RESV – will STOP"}}, // CONTROL_MODE
```
Comment (alfbr) [inline opm/simulators/utils/PartiallySupportedFlowKeywords.cpp:94]: Looks like this one fell between the cracks.

**Comment by alfbr** [comment]
Looks great! Only one fell between the cracks, correct that one and we should be good to go.

**Comment by OPMUSER** [comment]
Fixed.

Time to visit the Isles for a wee dram :-)

**Comment by alfbr** [comment]
Excellent, thank you! Personally, I have a weak spot for the single malts from Islay, but I'll join you :-)

**Comment by alfbr** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
Since all tests succeed for this one, it can be merged. @alfbr or @tskille please confirm. Are there still issues that should be marked critical but are not (because it makes cases fail that need to be fixed first)? Or is this the end of that?

**Comment by atgeirr** [comment]
I assume this should close #3837?

**Comment by tskille** [comment]
I would like to test this PR on a few full field models to see if there are any surprises.  I'll report back when I have done this.

**Comment by bska** [comment]
> I assume this should close #3837?

Did you mean #3737 instead?

**Comment by atgeirr** [comment]
> Did you mean #3737 instead?

Yes!

**Comment by tskille** [comment]
This is looking good and I hope this can be merged soon. 

When this PR is merged, I will add a new PR where I will include more updates to this file. These updates PR will also require some minor updates of two tests in opm-tests.

**Pull request merged**